js/gateio.js

```diff
@@ -2045,9 +2041,15 @@ module.exports = class gateio extends Exchange {
         const stopPrice = this.safeNumber (params, 'stopPrice');
         let methodTail = 'Orders';
         const reduceOnly = this.safeValue2 (params, 'reduce_only', 'reduceOnly');
-        const defaultTimeInForce = this.safeValue2 (params, 'tif', 'time_in_force', 'gtc');
+        const defaultTimeInForce = this.safeValue2 (params, 'tif', 'time_in_force');
         let timeInForce = this.safeValue (params, 'timeInForce', defaultTimeInForce);
-        params = this.omit (params, [ 'stopPrice', 'reduce_only', 'reduceOnly', 'tif', 'time_in_force', 'timeInForce' ]);
+        const postOnly = this.safeValue (params, 'postOnly', false);
+        if (postOnly) {
+            timeInForce = 'poc';
```

Comment:
According to gateio's docs, the `'poc'` value is not supported for conditional orders. Can you test that?

Reply:
where does it say that?

Reply:
Here (it does not list `poc`):

- https://www.gate.io/docs/developers/apiv4/en/#create-a-price-triggered-order


<img width="983" alt="Screenshot 2021-11-20 at 14 07 08" src="https://user-images.githubusercontent.com/1294454/142724059-18b4a613-f942-4e6b-9115-6f33a7be49dc.png">

- https://www.gate.io/docs/developers/apiv4/en/#create-a-price-triggered-order-2

<img width="984" alt="Screenshot 2021-11-20 at 14 06 14" src="https://user-images.githubusercontent.com/1294454/142724062-7bf33a70-06e0-465c-92aa-3b862a1ff260.png">

- https://www.gate.io/docs/developers/apiv4/en/#create-a-price-triggered-order-3

<img width="985" alt="Screenshot 2021-11-20 at 14 07 48" src="https://user-images.githubusercontent.com/1294454/142724063-363c77df-c200-4316-8d3c-ed8021729780.png">